plugins/admission/pkg/polaris/polaris.go

```diff
@@ -38,3 +42,31 @@ func GetPolarisReport(ctx context.Context, config polarisconfiguration.Configura
 	report.Contents = bytes
 	return report, nil
 }
+
+func GetPolarisValidateResults(kind string, decoder *admission.Decoder, req admission.Request, config polarisconfiguration.Configuration) (*validator.Result, error) {
+	var controller kube.GenericResource
+	var err error
+	if kind == "Pod" {
+		pod := corev1.Pod{}
+		err := decoder.Decode(req, &pod)
+		if err != nil {
+			return nil, err
+		}
+		if len(pod.ObjectMeta.OwnerReferences) > 0 {
+			logrus.Infof("Allowing owned pod %s/%s to pass through webhook", pod.ObjectMeta.Namespace, pod.ObjectMeta.Name)
```

Comment:
Does this mean that stand-alone pods would not be mutated, or is there additional significance (like admission-control)?

Reply:
I think that is the opposite... we do not process/mutate pods that have OwnerReference, meaning they are not standalone because they get covered in the else statement.